skeleton/webpack/aurelia_project/tasks/run.ext

```diff
@@ -22,19 +23,16 @@ const run = () => {
     args.push(host);
   }
 
-  return (new NPM()).run('start', ['--', ...args]);
+  return npm.run('start', ['--', ...args]);
 }
 
 const shutdownAppServer = () => {
   return new Promise(resolve => {
-    find('port', port)
-      .then(function (list) {
-        if (list.length) {
-          kill(list[0].pid, 'SIGKILL', function (err) {
+    if(npm.proc){
+      kill(npm.proc.pid, 'SIGKILL', function (err) {
```

Comment:
BTW, does the normal gentle kill `kill(npm.proc.pid, function (err) {` work too?

Reply:
I have to check it, shouldn't be any problem.

Reply:
It worked, so I will update it to gentle kill